ldk/go/readme.md

```diff
@@ -3,12 +3,12 @@
 ## Installation
 
 ```shell
-go get -u github.com/open-olive/loop-development-kit/ldk/go
+go get -u github.com/open-olive/loop-development-kit/ldk/go/v2
 ```
 
 ## Usage
 
-Full documentation available on [pkg.go.dev](https://pkg.go.dev/github.com/open-olive/loop-development-kit/ldk/go).
+Full documentation available on [pkg.go.dev](https://pkg.go.dev/github.com/open-olive/loop-development-kit/ldk/go/v2).
```

Comment:
My theory is that the CI/CD pipeline will generate a new (or update) `pkg.go.dev` package for ldk. Once this package adheres to the appropriate `/v2` module scheme, we can go back in and update projects that consume. (`examples`, `ldk-test`, `self-test-loop`)